Source/ACE/Entity/Creature.cs

```diff
@@ -4,22 +4,25 @@
 using ACE.Managers;
 using ACE.Network;
 using ACE.Network.Enum;
-using ACE.Network.GameAction;
 using ACE.Network.GameEvent.Events;
 using ACE.Network.Motion;
 using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
 
 namespace ACE.Entity
 {
     public class Creature : Container
     {
-        public CreatureAbility Strength {
+        public CreatureAbility Strength
+        {
             get { return AceObject.StrengthAbility; }
             set { AceObject.StrengthAbility = value; }
         }
 
+        public AceObject AceCopse
```

Comment:
I can't spell.   Crap.   To create a corpse - I need to basically clone the live body and create a new world object that is a container that looks like my cold dead corpse.   This was just a shortcut on my part - the only constructor I have for creating a world object - ie container class is from aceObject.   I can refactor that later when I have more time.